<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ASUS" name="Default test" tests="1" failures="0" timestamp="3 Dec 2017 17:55:46 GMT" time="4.1" errors="0">
  <testcase name="@BeforeMethod SetUp" time="0.0" classname="Application.App">
    <failure type="java.lang.NoSuchMethodError" message="org.openqa.selenium.remote.DesiredCapabilities.edge()Lorg/openqa/selenium/remote/DesiredCapabilities;">
      <![CDATA[java.lang.NoSuchMethodError: org.openqa.selenium.remote.DesiredCapabilities.edge()Lorg/openqa/selenium/remote/DesiredCapabilities;
	at org.openqa.selenium.edge.EdgeOptions.toCapabilities(EdgeOptions.java:98)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:98)
	at Utilities.UtilCommon.OpenBrowser(UtilCommon.java:64)
	at Application.App.SetUp(App.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod SetUp -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="Application.App">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="TC001_Login" time="0.005" classname="Application.App">
    <skipped/>
  </testcase> <!-- TC001_Login -->
</testsuite> <!-- Default test -->
